intel_extension_for_transformers/llm/runtime/graph/application/main_pybind.cpp

```diff
@@ -350,40 +395,40 @@ std::vector<std::vector<model_token>> Model::generate_tokens(const std::vector<s
     return rets;
   }
 
-  if (curr_input_ids.empty()) {
+  if (curr_input_ids[0].empty()) {
```

Comment:
make numbers static and meanful as much as you can

```suggestion
#define ID_HEAD 0
  if (curr_input_ids[ID_HEAD].empty()) {
```

Reply:
updated in https://github.com/intel/intel-extension-for-transformers/pull/700/commits/18842dee2c880511621993175ff5372a98c5554b